library/generatedmodel/src/main/java/com/amplifyframework/datastore/generated/model/Caretaker.java

```diff
@@ -0,0 +1,126 @@
+package com.amplifyframework.datastore.generated.model;
+
+
+import androidx.core.util.ObjectsCompat;
+
+/** This is an auto generated class representing the Caretaker type in your schema. */
```

Comment:
Not a blocker, but what do you think of moving generated files to gitignore (since everyone can build them on their machine) and adding building this to gradle steps. This could reduce the number of unnecessary merge conflicts.